holoviews/tests/operation/test_operation.py

```diff
@@ -179,6 +185,41 @@ def test_dataset_weighted_histogram_dask(self):
         self.assertIsInstance(op_hist.data['y'], da.Array)
         self.assertEqual(op_hist, hist)
 
+    @ibis_skip
+    @pytest.mark.usefixtures('ibis_sqlite_backend')
+    def test_dataset_histogram_ibis(self):
+        df = pd.DataFrame(dict(x=np.arange(10)))
+        t = ibis.memtable(df, name='t')
+        ds = Dataset(t, vdims='x')
+        op_hist = histogram(ds, dimension='x', num_bins=3, normed=True)
+
+        hist = Histogram(([0, 3, 6, 9], [0.1, 0.1, 0.133333]),
+                         vdims=('x_frequency', 'Frequency'))
+        self.assertEqual(op_hist, hist)
+
+    @ibis_skip
+    @pytest.mark.usefixtures('ibis_sqlite_backend')
+    def test_dataset_cumulative_histogram_ibis(self):
+        df = pd.DataFrame(dict(x=np.arange(10)))
+        t = ibis.memtable(df, name='t')
+        ds = Dataset(t, vdims='x')
+        op_hist = histogram(ds, num_bins=3, cumulative=True, normed=True)
+
+        hist = Histogram(([0, 3, 6, 9], [0.3, 0.6, 1]),
+                         vdims=('x_frequency', 'Frequency'))
+        self.assertEqual(op_hist, hist)
+
+    def test_dataset_histogram_explicit_bins_ibis(self):
```

Comment:
I think you forgot or didn't push the decorators for this test.

Reply:
Damned.